src/data-workspace/validation/validation-rule-expression.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ValidationRuleViolationWithMetaDataPropTypes } from './validation-result-prop-types.js'
+
+const ValidationRuleExpression = ({ validationRule }) => {
+    const {
+        leftsideValue,
+        rightsideValue,
+        metaData: { leftSide, rightSide, operator } = {},
+    } = validationRule
+
+    // @ToDO: confirm the human readable equivalent for the operators
+    const operatorDisplay = {
+        equal_to: '==',
+        not_equal_to: '!=',
+        greater_than: '>',
+        greater_than_or_equal_to: '>=',
+        less_than: '<',
+        less_than_or_equal_to: '<=',
+        compulsory_pair: 'is compulsory with',
+        exclusive_pair: 'is exclusive to',
+    }
+    return (
+        <>
+            {leftSide.displayDescription} ({leftsideValue}){' '}
+            {operatorDisplay[operator] ?? operator}{' '}
+            {rightSide.displayDescription} ({rightsideValue})
+        </>
+    )
```

Comment:
Component can return strings, there's no need to wrap this within a `Fragment`.

Reply:
> Currently the sidebar is closing when changing a field. You can ....In the specs it says that when changing a value in the form, the following should be displayed

This is tackled in a different subtask / PR: https://dhis2.atlassian.net/browse/TECH-1275


> There's too much spacing below the error message when an api error occurs:

updated ... looks like this
<img width="373" alt="image" src="https://user-images.githubusercontent.com/1014725/181201994-caed0fc1-33c2-4282-94c9-106f5a5e7eb3.png">


> Component can return strings, there's no need to wrap this within a Fragment.
updated